algos/src/sample_rate_conversion.rst

```diff
@@ -266,52 +265,48 @@ parameters help:
 		./testbench -h
 
 The executable can be debugged with any C debugger/IDE tool and any
-code analysis tools such as valgrind and gprof. Some tips for
-debugging:
+code analysis tool such as valgrind and gprof. Some tips for
+debugging are:
 
-- In interactive debugging it can be useful to remove the default -O2
-  optimization to get linear stepping of code lines and accurate
+- In interactive debugging, it can be useful to remove the default -O2
+  optimization in order to get linear stepping of code lines and accurate
   breakpoints.
-      
-- In gdb based debuggers when debugging audio processing it can be
-  useful to plot with gnuplot vectors of numerical values as
-  graphs. Instructions for setting it up is available in
-  https://sourceware.org/gdb/wiki/PlottingFromGDB .
-
 
+- When debugging audio processing in gdb-based debuggers, it can be useful
```

Comment:
@singalsu Please review this paragraph, beginning at line 275.

Reply:
It's good